src/IRtext.cpp

```diff
@@ -1,261 +1,262 @@
+const PROGMEM char* kHoursStr = D_STR_HOURS;
+const PROGMEM char* kMinuteStr = D_STR_MINUTE;
+const PROGMEM char* kMinutesStr = D_STR_MINUTES;
+const PROGMEM char* kSecondStr = D_STR_SECOND;
+const PROGMEM char* kSecondsStr = D_STR_SECONDS;
+const PROGMEM char* kNowStr = D_STR_NOW;
+const PROGMEM char* kThreeLetterDayOfWeekStr = D_STR_THREELETTERDAYS;
+
+const PROGMEM char* kYesStr = D_STR_YES;
+const PROGMEM char* kNoStr = D_STR_NO;
+const PROGMEM char* kTrueStr = D_STR_TRUE;
+const PROGMEM char* kFalseStr = D_STR_FALSE;
+
+const PROGMEM char* kRepeatStr = D_STR_REPEAT;
+const PROGMEM char* kCodeStr = D_STR_CODE;
+const PROGMEM char* kBitsStr = D_STR_BITS;
+
+// Protocol Names
+// Needs to be in decode_type_t order.
+const PROGMEM char *kAllProtocolNamesStr =
+    D_STR_UNUSED "\x0"
+    D_STR_RC5 "\x0"
+    D_STR_RC6 "\x0"
+    D_STR_NEC "\x0"
+    D_STR_SONY "\x0"
+    D_STR_PANASONIC "\x0"
+    D_STR_JVC "\x0"
+    D_STR_SAMSUNG "\x0"
+    D_STR_WHYNTER "\x0"
+    D_STR_AIWA_RC_T501 "\x0"
+    D_STR_LG "\x0"
+    D_STR_SANYO "\x0"
+    D_STR_MITSUBISHI "\x0"
+    D_STR_DISH "\x0"
+    D_STR_SHARP "\x0"
+    D_STR_COOLIX "\x0"
+    D_STR_DAIKIN "\x0"
+    D_STR_DENON "\x0"
+    D_STR_KELVINATOR "\x0"
+    D_STR_SHERWOOD "\x0"
+    D_STR_MITSUBISHI_AC "\x0"
+    D_STR_RCMM "\x0"
+    D_STR_SANYO_LC7461 "\x0"
+    D_STR_RC5X "\x0"
+    D_STR_GREE "\x0"
+    D_STR_PRONTO "\x0"
+    D_STR_NEC_LIKE "\x0"
+    D_STR_ARGO "\x0"
+    D_STR_TROTEC "\x0"
+    D_STR_NIKAI "\x0"
+    D_STR_RAW "\x0"
+    D_STR_GLOBALCACHE "\x0"
+    D_STR_TOSHIBA_AC "\x0"
+    D_STR_FUJITSU_AC "\x0"
+    D_STR_MIDEA "\x0"
+    D_STR_MAGIQUEST "\x0"
+    D_STR_LASERTAG "\x0"
+    D_STR_CARRIER_AC "\x0"
+    D_STR_HAIER_AC "\x0"
+    D_STR_MITSUBISHI2 "\x0"
+    D_STR_HITACHI_AC "\x0"
+    D_STR_HITACHI_AC1 "\x0"
+    D_STR_HITACHI_AC2 "\x0"
+    D_STR_GICABLE "\x0"
+    D_STR_HAIER_AC_YRW02 "\x0"
+    D_STR_WHIRLPOOL_AC "\x0"
+    D_STR_SAMSUNG_AC "\x0"
+    D_STR_LUTRON "\x0"
+    D_STR_ELECTRA_AC "\x0"
+    D_STR_PANASONIC_AC "\x0"
+    D_STR_PIONEER "\x0"
+    D_STR_LG2 "\x0"
+    D_STR_MWM "\x0"
+    D_STR_DAIKIN2 "\x0"
+    D_STR_VESTEL_AC "\x0"
+    D_STR_TECO "\x0"
+    D_STR_SAMSUNG36 "\x0"
+    D_STR_TCL112AC "\x0"
+    D_STR_LEGOPF "\x0"
+    D_STR_MITSUBISHI_HEAVY_88 "\x0"
+    D_STR_MITSUBISHI_HEAVY_152 "\x0"
+    D_STR_DAIKIN216 "\x0"
+    D_STR_SHARP_AC "\x0"
+    D_STR_GOODWEATHER "\x0"
+    D_STR_INAX "\x0"
+    D_STR_DAIKIN160 "\x0"
+    D_STR_NEOCLIMA "\x0"
+    D_STR_DAIKIN176 "\x0"
+    D_STR_DAIKIN128 "\x0"
+    D_STR_AMCOR "\x0"
+    D_STR_DAIKIN152 "\x0"
+    D_STR_MITSUBISHI136 "\x0"
+    D_STR_MITSUBISHI112 "\x0"
+    D_STR_HITACHI_AC424 "\x0"
+    D_STR_SONY_38K "\x0"
+    D_STR_EPSON "\x0"
+    D_STR_SYMPHONY "\x0"
+    D_STR_HITACHI_AC3 "\x0"
+    D_STR_DAIKIN64 "\x0"
+    D_STR_AIRWELL "\x0"
+    D_STR_DELONGHI_AC "\x0"
+    D_STR_DOSHISHA "\x0"
+    D_STR_MULTIBRACKETS "\x0"
+    D_STR_CARRIER_AC40 "\x0"
+    D_STR_CARRIER_AC48 "\x0"

```

Comment:
Same as with the enum

Reply:
Correct. The order has to be the same as `decode_type_t`s, and the suggested (new) order needs to be fixed, and thus, so will this to match it.